manifests/repo/debian.pp

```diff
@@ -47,7 +47,7 @@
     source => $key['source'],
   }})
 
-  ::apt::source { "source_php_${release}":
+  ::apt::source { 'php-upstream':
```

Comment:
@bastelfreak we should make this name configureable. i personally prefer to name my repo files according the upstream name `dotdeb` or `sury` and not according the content `php`.  both provide php but also other things. especially dotdeb provides a lot of other stuff

Reply:
I agree, but thats out of the scope here.